packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -131,6 +132,14 @@ class TestCommand extends FlutterCommand with DeviceBasedDevelopmentArtifacts {
         defaultsTo: 'coverage/lcov.info',
         help: 'Where to store coverage information (if coverage is enabled).',
       )
+      ..addMultiOption('coverage-package',
+        help: 'A regular expression matching packages names '
```

Comment:
I think regular expressions here is surprising. Since this is a multi-option, why not just use multiple package names?

Reply:
As I wrote in the Discord, this is to simplify matching of the multiple packages and avoid editing ci/cd scripts for the coverage collection each time new package is introduced.  

Additionally, with the current solution it is possible to report **each** package that was discovered during testing using the `.*` regular expression. Even though I personally can't think of a use case atm, but I don't understand why we have to limit this functionality.

Reply:
Ok, I agree, I think we probably need regexes here.